### ISSUE TITLE
Clicking on a Link closes Chrome without warnings (Version 0.5.7)

### ISSUE THREAD

**Issue by figuretti** [zhukov/webogram#1450]
Like the title said... if i try to open a link received on the webapp, it closes the app + Chrome without warnings

**Comment by zhukov** [comment]
@figuretti Yep, was fixed in 0.5.7.1, which is already shipped to the store. It's some chrome bug, related to noreferrer option for links.

**Comment by Peque** [comment]
@zhukov But those changes are not pushed to the repo, are they? :sweat:

**Comment by zhukov** [comment]
@Peque Yep, it was hotfix. Will push soon

**Comment by DaniilVystoskiy** [comment]
The problem is still there, but with Telegram desktop app. Countered crashes two times for the last two days. Moreover, Telegram app crashes too.
Using the latest version- 0.5.7.2, Chrome v.59.0.3071.115

**Comment by Peque** [comment]
@DaniilVystoskiy Do you have webogram or Telegram desktop? (If the later, you should report the issue in [`tdesktop`](https://github.com/telegramdesktop/tdesktop)). Also, version 0.5.7.2 does not exist (?).

@zhukov for me everything seems to work fine after the hot-fix (currently at [`8003e73`](https://github.com/zhukov/webogram/tree/8003e73015024a9a05865c2db0891891d7f8cefe)).

**Comment by DaniilVystoskiy** [comment]
@Peque yes, it's webogram (Telegram Web v. 0.5.7.1, as "About" section says). Sorry for the typo in version number. There is a link to this particular GitHub, this is how I get here.

As I get it, the issue doesn't spread on links, that were proceeded through Telegram API and got previews section, but thouse links, that are just simple underlined blue links, causes instant crashing both Telegram and Chrome.

**Comment by zhukov** [comment]
@DaniilVystoskiy can you record a video?

**Comment by DaniilVystoskiy** [comment]
@zhukov huh, I've checked again that particular link, that was causing crash, and for now everything is working good.

I will keep watching for the bug, and if counter it again, will try to record it.

**Comment by DaniilVystoskiy** [comment]
@zhukov I recorded it! Just before start to work, I usually check Telegram channels on daily basis. I turned on the recorder and took my hot cup of coffee... the very first link I clicked in chat causes the crash. 
[Check it here on 20 sec](https://www.dropbox.com/s/fz9osu4h53ivf7x/2017-07-07%2009-40-46.mp4?dl=0)
(Sorry for video quality, but it is enough to understand what has happened).

After relaunching both Telegram and Chrome, everything is ok.
